### DISCUSSION TITLE
Any plans to expose the @milkdown/components package for general usage?

### DISCUSSION THREAD

**Discussion by martinpokorny** [Milkdown/milkdown#1149] [General]
Hi,

I couldn't find anything about it in the docs, but I found PR from August 2023 that merged in the `@milkdown/components/list-item-block component`, which by the looks of it is exactly what I need. However when I tried to import it based on example provided in `/e2e/`, I could not get it working.

```
[commonjs--resolver] Missing "./list-item-block" specifier in "@milkdown/components" package
```

As far as I am aware, it appears that components are probably not meant to be used by us, at least not yet... is that correct? And if that so, are there any plans to change it?

Thank you for your time. :)

**Comment by Saul-Mirone** [comment]
I'll release a new version this weekend and you'll be able to use it at that time.
